src/infra/metrics.rs

```diff
@@ -147,6 +153,24 @@ fn into_labels(labels: Vec<(&'static str, LabelValue)>) -> Vec<MetricsLabel> {
         .collect()
 }
 
+/// Labels for storage types
+
+pub enum StorageType {
```

Comment:
I don't think it makes sense to have this here or anywhere.
This module does not have any specifics enums from a specific context.

You can put in the in StratusStorage module, but I think just having a few constants there like "temp_storage", "perm_storage" and "default_value" should be enough.

Reply:
Sure, I'll do that. 

As a side note: Shouldn't the constants be in this module either way? These labels don't concern the storage module, they are only relevant in the context of metrics. Sure this module does not have specific labels, but is there a reason that it shouldn't?